pkg/serviceaccount/validation/validation.go

```diff
@@ -63,3 +65,22 @@ func ValidateDescription(val interface{}) error {
 
 	return errors.New(`invalid service account description; only alphanumeric characters and "-", ".", "," are accepted`)
 }
+
+// ValidateUUID validates if ID is a valid UUID
+func ValidateUUID(localizer localize.Localizer) func(v interface{}) error {
```

Comment:
I think it will be a good idea for readibility if we do not use anonymous return functions anymore for all validators. I'll explain in a follow up issue

Reply:
> PS: It would also be nice to validate the flag for UUId like you had suggested.

It has been done in `reset_credentials.go`. Did you expect something else?

Reply:
> I think it will be a good idea for readibility if we do not use anonymous return functions anymore for all validators. I'll explain in a follow up issue

Maked #721 as WIP, it has localizations for validators using anonymous functions, should this PR wait for the issue?

Reply:
No you can merge this PR beforehand if you like.